frontend/src/pages/instance/components/InstanceLogs.vue

```diff
@@ -101,7 +100,9 @@ export default {
     },
     methods: {
         shouldPoll: function () {
-            return this.$route.fullPath.match(/\/(?:application|instance)\/[^/]+\/logs/)
+            return Object.hasOwnProperty.call(this.$route, 'meta') &&
```

Comment:
why not `this.$route.hadsOwnProperty('meta')`?

Reply:
Force of habit mainly, but also protects against this.$route being undefined or null